detection_rules/rule.py

```diff
@@ -343,7 +343,7 @@ class RelatedIntegrations:
     exceptions_list: Optional[list]
     license: Optional[str]
     false_positives: Optional[List[str]]
-    filters: Optional[List[Filter]]
+    filters: Optional[List[dict]]
```

Comment:
I was thinking about removing the `Filter` class definitions as well, but I thought we should leave it until we have time to decide on whether to implement the extended classes or not at all.

Reply:
I agree, I think leaving it in is a good idea. 

NIT. If we leave it in we may want to add a docstring comment to remind us where is was used/why we have it. 

```python

@dataclass(frozen=True)
class Filter:
    """Kibana Filter for Base Rule Data. Currently unused."""
    
```

<details><summary>Patch contents</summary>
<p>

```patch

diff --git a/detection_rules/rule.py b/detection_rules/rule.py
index cd81fc19..46edee8c 100644
--- a/detection_rules/rule.py
+++ b/detection_rules/rule.py
@@ -310,6 +310,7 @@ class Query:
 
 @dataclass(frozen=True)
 class Filter:
+    """Kibana Filter for Base Rule Data. Currently unused."""
     meta: FilterMeta
     state: Optional[FilterStateStore] = field(metadata=dict(data_key="$state"))
     query: Optional[Union[Query, Dict[str, Any]]] = None

```

</p>
</details>